client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-list-panel.tsx

```diff
@@ -46,7 +53,22 @@ const PatternListPanel = ( {
 				onSelect={ onSelect }
 				selectedPattern={ selectedPattern }
 				selectedPatterns={ selectedPatterns }
+				isShowMorePatterns={ isShowMorePatterns }
 			/>
+			{ ! isShowMorePatterns && (
```

Comment:
I guess we have to check whether there are patterns that are not priority patterns with the current category. If not, we should not show this button.

Reply:
✅ Done, thanks